### ISSUE TITLE
[Request] Add Green/Red color to "like bar" on video page

### ISSUE THREAD

**Issue by deathdragon1987** [elliotwaite/youtube-thumbnail-rating-bar#3]
![capture](https://user-images.githubusercontent.com/6306835/42085005-2a98b824-7b87-11e8-8440-3d7cff3f88ad.PNG)

Change to this, if possible

![capture1](https://user-images.githubusercontent.com/6306835/42085111-6aa693c8-7b87-11e8-9b86-0f75ebaff2db.PNG)

**Comment by petersori** [comment]
This has been sorted out allready.
Please click on Extension icon and you'll get the menu to choose from classic and modern.

**Comment by elliotwaite** [comment]
Ah, I had only made that setting change the colors for the thumbnail rating bar. I didn't think to make it also change the colors for the video page rating bar. Thanks for the suggestion. I just released an update so it does now.

However, the picture you included for the green and red bar has a thicker height than the blue and gray bar and has rounded corners and a slight shadow. I tried adding those stylings, but to me, I thought it looked better to just keep it thin and only changing the colors to green and red. Let me know what you think.
